src/APIFunctions/3DPrinting.js

```diff
@@ -1,10 +1,10 @@
 import axios from 'axios';
 import { ApiResponse } from './ApiResponses';
 let config = require('../config/config.json');
-let GENERAL_API_URL = process.env.NODE_ENV === 'production' ?
-  config.GENERAL_API_URL_PROD : config.GENERAL_API_URL;
-let RPC_API_URL = process.env.NODE_ENV === 'production' ?
-  config.RPC_API_URL_PROD : config.RPC_API_URL;
+let GENERAL_API_URL =
+  process.env.NODE_ENV === 'production'
+    ? config.GENERAL_API_URL_PROD
+    : config.GENERAL_API_URL;
```

Comment:
it appears my autoformatter is correcting a bunch of stuff. i will look into this

Reply:
turn it off turn it off